tests/test_pattern_rewriter.py

```diff
@@ -293,6 +293,8 @@ def match_and_rewrite(mod: ModuleOp, rewriter: PatternRewriter):
         new_cst = Constant.from_int_and_width(42, i32)
         rewriter.insert_op_at_pos(new_cst, mod.regions[0].blocks[0], 0)
 
+    a: str = 1
```

Comment:
looks like tests aren't supported, or we'd need to set up pyright to run on tests also, can be done in follow-up PRs